### PR TITLE
New package: LabyMediaGmbH.LabyModLauncher version 1.0.19

### PR DESCRIPTION
**Pull request by CanePlayz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111685&drop=dogfoodAlpha

### CODE REVIEW

**Comment by CanePlayz** [comment]
What evidence should I provide? Here's their website with a download button: https://labymod.net/de/download

File: manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LabyMediaGmbH.LabyModLauncher
+PackageVersion: 1.0.19
+PackageLocale: en-US
+Publisher: LabyMedia GmbH
```
Comment (BrandonWanHuanSheng) [inline manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.locale.en-US.yaml:7]:
```suggestion
Publisher: LabyMedia GmbH
PublisherUrl: https://labymod.net
PackageUrl: https://labymod.net/en/download
```

**Review by BrandonWanHuanSheng — CHANGES_REQUESTED** [review]
It look like you can make the manifest more completed by adding PackageUrl and PublisherUrl.

**Comment by CanePlayz** [comment]
> It look like you can make the manifest more completed by adding PackageUrl and PublisherUrl.

Thanks!

**Comment by CanePlayz** [comment]
Hmm, that didn't help unfortunately

**Comment by BrandonWanHuanSheng** [comment]
I think later on someone is assigned on the pull request will add this label https://github.com/microsoft/winget-pkgs/labels/Waived-Validation-Domain

**Comment by CanePlayz** [comment]
> I think later on someone is assigned on the pull request will add this label https://github.com/microsoft/winget-pkgs/labels/Waived-Validation-Domain

I see, thanks

File: manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LabyMediaGmbH.LabyModLauncher
+PackageVersion: 1.0.19
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://laby-releases.s3.de.io.cloud.ovh.net/launcher/win32/x64/LabyModLauncherSetup-latest.exe
```
Comment (stephengillie) [inline manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml:9]:
```suggestion
  InstallerUrl: https://releases.r2.labymod.net/launcher/win32/x64/LabyModLauncherSetup-latest.exe
```

**Comment by CanePlayz** [comment]
Thanks Stephen!

**Comment by CanePlayz** [comment]
Hmm, I get this exact hash

**Comment by BrandonWanHuanSheng** [comment]
#111634

**Comment by BrandonWanHuanSheng** [comment]
That pipelines was unstable.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ##[error] Installer https://releases.r2.labymod.net/launcher/win32/x64/LabyModLauncherSetup-latest.exe
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   9274587020718C3AB64B013A0E18822526C53EA687F29840A340920345CB68C4
Expected hash: 6cdca60032796c809a1da444939ff2c01f2345497ecd0d0807ab04aa28e72190

**Comment by stephengillie** [comment]
> @stephengillie Where did you get that link from?

It was the download link on the PackageUrl.

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 7:27:13 PM:  > 2023-07-10 19:27:12.318 [CLI ] Package hash verification failed. SHA256 in manifest [6cdca60032796c809a1da444939ff2c01f2345497ecd0d0807ab04aa28e72190] does not match download [9274587020718c3ab64b013a0e18822526c53ea687f29840a340920345cb68c4]

File: manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LabyMediaGmbH.LabyModLauncher
+PackageVersion: 1.0.19
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://releases.r2.labymod.net/launcher/win32/x64/LabyModLauncherSetup-latest.exe
+  InstallerSha256: 6CDCA60032796C809A1DA444939FF2C01F2345497ECD0D0807AB04AA28E72190
```
Comment (stephengillie) [inline manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml:10]:
```suggestion
  InstallerSha256: 9274587020718c3ab64b013a0e18822526c53ea687f29840a340920345cb68c4
```

**Comment by CanePlayz** [comment]
Why they changin the hash every 2 minutes 😭

**Review by stephengillie — APPROVED** [review]

**Pull request merged**